soh/soh/OTRGlobals.cpp

```diff
@@ -884,6 +884,16 @@ std::shared_ptr<Ship::Resource> ResourceMgr_LoadResource(const char* path) {
     return OTRGlobals::Instance->context->GetResourceManager()->LoadResource(Path.c_str());
 }
 
+extern "C" char* GetResourceDataByNameHandlingMQ(const char* path) {
+    auto res = GetResourceByNameHandlingMQ(path);
```

Comment:
got a little worried for a second before i noticed these didn't have the same name
`GetResourceByNameHandlingMQ`
vs
`GetResourceDataByNameHandlingMQ`

this makes sense